### DISCUSSION TITLE
Pop-up when hovering on nav. (origin theme)

### DISCUSSION THREAD

**Discussion by Jicknones** [swellstores/community#567] [Help] [answered]
Anyone know how to get rid of this pop-up when hovering? I deleted all these products and categories, still shows for some reason? (contacted support, sent me here)

<img width="1259" alt="Screenshot 2024-08-23 at 8 16 25 AM" src="https://github.com/user-attachments/assets/89819852-c235-4e66-b356-8cc00b09ddc2">

**Comment by logeshswell** [comment]
@Jicknones It looks like an issue in the current version of your theme's source code. To resolve it, you need to pull the latest changes of origin theme into your source code.

**Reply by logeshswell** [reply]
Currently, we are facing a glitch in pulling the storefront code. I will let you know once it is resolved and then you can try it out.

**Reply by logeshswell** [reply, marked as answer]
The issue with pulling the storefront code is fixed now.
I also upgraded your storefront code to the latest version of origin-theme and the menu navigation is working fine now.
Please check and confirm.

**Comment by Jicknones** [comment]
Hi Logesh—Thank you!

It still is not showing all of the products, is there anything I can do on
my end to fix this?

Thanks for your help!

**Reply by logeshswell** [reply]
Only active products are returned in the frontend (or storefront) APIs.
